envs/environment.yaml

```diff
@@ -2,19 +2,19 @@
 #
 # SPDX-License-Identifier: MIT
 
-name: pypsa-eur
+name: pypsa-eur-nov14
 channels:
   - conda-forge
   - bioconda
   - http://conda.anaconda.org/gurobi
 dependencies:
-  - python>=3.8
+  - python=3.8 # fix because gurobi has no builds for python 3.9
```

Comment:
Don't think this is necessary. I have python 3.9 and gurobi works smoothly (however cplex is not available on python 3.9)

Reply:
Interesting. It works. But why if the builds for py39 are not in the conda channel? https://anaconda.org/Gurobi/gurobi/files